.circleci/config.yml

```diff
@@ -1031,7 +1030,7 @@ commands:
           # by default, electron-builder will NOT sign app built in a pull request
           # even our internal one (!)
           # Usually this is not a problem, since we only build and test binary
-          # built on "develop" and "master" branches
+          # built on "develop" branches
```

Comment:
Verbiage suggestion:
'Built on the `develop` branch'

Reply:
ohh good catch